src/index.ts

```diff
@@ -156,6 +159,9 @@ export const hostPortsVar = (
 	}
 	return hostPairs.split(',').map((hostPair): HostPort => {
 		const [host, maybePort] = hostPair.trim().split(':');
+		if (maybePort == null) {
```

Comment:
I guess it could also be checking the `host` here?

Reply:
Makes sense, added.